quickstart-node/fetch-schema.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+  curl "https://api.monday.com/v2/get_schema?format=sdl&version=stable" -o src/schema.graphql
```

Comment:
since you pass version 2024-10 down, should we just make a const and use it here as well?

Reply:
you mean do `...&version=2024-10...`?

Reply:
but isn't it a problem in the sdk that it does not do it automatically? Theoretically I should not touch this file right?